felix/fv/bpf_test.go

```diff
@@ -4390,6 +4382,10 @@ func describeBPFTests(opts ...bpfTestOpt) bool {
 			})
 
 			It("should have connectivity when DNAT redirects to-host traffic to a local pod.", func() {
+				if NFTMode() {
+					Skip("NFT does not support third-party rules")
```

Comment:
I think this is meant to test what happens if the user creates their own NAT rules, which they could do in NFT mode in a separate table?